File: testprojects/master_python_mainone/pipelines/DONOT_OPEN_PYTHON/code/job/graph/SchemaTransform_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from job.config.ConfigStore import *
from job.udfs.UDFs import *

def SchemaTransform_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0\
        .withColumn("concat short int", concat(col("`c   short  --`"), col("`c-int-column type`")))\
        .drop("c- - -double")\
        .withColumnRenamed("c-decimal", "c-decimal renamed")

File: testprojects/master_scala_mainone/pipelines/ScalaGCP/code/src/main/scala/graph/SchemaTransform_1.scala
package graph

import io.prophecy.libs._
import config.ConfigStore._
import udfs.UDFs._
import udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import java.time._

object SchemaTransform_1 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.withColumn("c_concat_new_short_decimal",
                  concat(col("`c_struct-c_short`"), col("`c_struct-c_decimal`"))
      )
      .drop("c_array_long")
      .withColumnRenamed("c_array_boolean", "c_array_boolean_renamed")

}
